open-api/rest-catalog-open-api.yaml

```diff
@@ -1972,8 +1972,8 @@ components:
             operation:
               type: string
               enum: ["append", "replace", "overwrite", "delete"]
-            additionalProperties:
-              type: string
+          additionalProperties:
```

Comment:
this change LGTM and also conforms to https://swagger.io/docs/specification/data-models/dictionaries/